repositories/dbmodels/db_organization_feature_access.go

```diff
@@ -0,0 +1,65 @@
+package dbmodels
+
+import (
+	"time"
+
+	"github.com/checkmarble/marble-backend/models"
+	"github.com/checkmarble/marble-backend/utils"
+)
+
+const TABLE_ORGANIZATION_FEATURE_ACCESS = "organization_feature_access"
+
+var SelectOrganizationFeatureAccessColumn = utils.ColumnList[models.OrganizationFeatureAccess]()
```

Comment:
```suggestion
var SelectOrganizationFeatureAccessColumn = utils.ColumnList[DBOrganizationFeatureAccess]()
```
should be this instead (the "db" tags are used to map the field names, and of course the exact list of fields may differ between the DB DTO and the model)